redpanda/datadog_checks/redpanda/redpanda.py

```diff
@@ -0,0 +1,59 @@
+from six.moves.urllib.parse import urlparse
+
+from datadog_checks.base import ConfigurationError, OpenMetricsBaseCheck
+
+from .metrics import ADDITIONAL_METRICS_MAP, INSTANCE_DEFAULT_METRICS
+
+
+class RedpandaCheck(OpenMetricsBaseCheck):
```

Comment:
Hi @rkruze ! Unfortunately we haven't had a chance to update the docs to reflect the step by step on using OpenmetricsbaseCheckV2, however it's usage is fairly similar.

The conf.yaml.example options look like so: https://github.com/DataDog/integrations-core/blob/master/openmetrics/datadog_checks/openmetrics/data/conf.yaml.example

Most importantly the biggest change is using `openmetrics_endpoint` instead of `prometheus_url`.

It's important to note that users need to be on Agent 7.26.x or higher and be on Python 3. There are significant performance improvements and accurate metric type submission.

Reply:
Done

Reply:
Hi @rkruze, following up on your documentation request, Christine recently updated our public docs with information on writing an OpenMetricsv2 check: https://docs.datadoghq.com/developers/custom_checks/prometheus/.